shell/content/docs/en-us/whats-new.md

```diff
@@ -2,6 +2,33 @@
 title: What's New in 2.6
 ---
 
+
+## New in 2.6.7
+
+### Amazon Billing in the Marketplace
+
+- Now you can purchase support for Rancher through the AWS marketplace 
+
+### Ability to configure Azure AD with MSAL
+
+- [need description]
```

Comment:
```
Microsoft have deprecated the Azure AD Graph API that Rancher had been using for authentication via Azure AD. 
A configuration update is necessary to make sure users can still use Rancher with Azure AD.
For more details see: https://rancher.com/docs/rancher/v2.6/en/admin-settings/authentication/azure-ad/#migrating-from-azure-ad-graph-api-to-microsoft-graph-api
```

Taken and modified from: https://github.com/rancher/rancher/issues/29306#issuecomment-1195846938

Reply:
Thanks!